src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAsyncCollector.cs

```diff
@@ -6,9 +6,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Azure.EventHubs;
-using Microsoft.Azure.ServiceBus;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.EventHubs
```

Comment:
Are you thinking you want to take this breaking namespace change?

Reply:
Nope, just juggling a lot of things and it slipped my mind. Thanks :)